mypy/subtypes.py

```diff
@@ -155,19 +150,18 @@ def visit_instance(self, left: Instance) -> bool:
                        for lefta, righta, tvar in
                        zip(t.args, right.args, right.type.defn.type_vars))
         if isinstance(right, TypeType):
-            item = right.item
-            if isinstance(item, TupleType):
-                item = item.fallback
```

Comment:
Why did you remove the case of ``TupleType``? A named tuple can appear in ``Type[]``.